src/functions.js

```diff
@@ -324,10 +324,48 @@ var _ = Mavo.Functions = {
 
 	len: text => str(text).length,
 
+	/**
+     * Search if a group, collection, or primitive contains needle
+	 * @returns Boolean if a haystack of object or primitive is passed
+	 * @returns Array of booleans if a haystack of array is passed
+     */
+    contains: (haystack, needle) => {
+		var ret = Mavo.Script.binaryOperation(haystack, needle, {
+			scalar: (haystack, needle) => {
+				var type = $.type(haystack);
+				if (type === "object") {
```

Comment:
We don’t need the `type` variable here anymore, we're only using this once. The whole point of the variable was to cache the results so we don't have to run the function twice.